rapid/workflow/data/models.py

```diff
@@ -128,7 +128,7 @@ class PipelineInstance(BaseModel, DateModel, Base):
     priority = Column(Integer, default=0, index=True)
 
     stage_instances = relationship("StageInstance", backref="pipeline_instance", order_by="asc(StageInstance.order)")
-    action_instances = relationship("ActionInstance", order_by="ActionInstance.id")
+    action_instances = relationship("ActionInstance", order_by="ActionInstance.id", overlaps='pipeline_instance')
```

Comment:
This was a new one for me. SQLAlchemy never ceases to amaze. Or confuse. Or both.